scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -837,77 +840,6 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
       (isCaseClassOrObject || (token.is[KwCase] && token.next.is[Ident] && dialect.allowEnums))
   }
 
-  @classifier
-  trait SkAs {
-    val name = "as"
-    def unapply(token: Token): Boolean = {
-      isSoftKw(token, name) && dialect.allowAsPatternBinding
-    }
-  }
-  @classifier
-  trait SkUsing {
-    val name = "using"
-    def unapply(token: Token): Boolean = {
-      isSoftKw(token, name) && dialect.allowGivenUsing
-    }
-  }
-
-  @classifier
-  trait SkInline {
-    val name = "inline"
-    def unapply(token: Token): Boolean = {
-      isSoftKw(token, name) && dialect.allowInlineMods
-    }
-  }
-
-  @classifier
-  trait SkOpaque {
-    val name = "opaque"
-    def unapply(token: Token): Boolean = {
-      isSoftKw(token, name) && dialect.allowOpaqueTypes
-    }
-  }
-
-  @classifier
-  trait SkOpen {
-    val name = "open"
-    def unapply(token: Token): Boolean = {
-      isSoftKw(token, name) && dialect.allowOpenClass
-    }
-  }
-
-  @classifier
-  trait SkTransparent {
-    val name = "transparent"
-    def unapply(token: Token): Boolean = {
-      isSoftKw(token, name) && dialect.allowInlineMods
-    }
-  }
-
-  @classifier
-  trait SkDerives {
-    val name = "derives"
-    def unapply(token: Token): Boolean = {
-      isSoftKw(token, name) && dialect.allowDerives
-    }
-  }
-
-  @classifier
-  trait SkEnd {
-    val name = "end"
-    def unapply(token: Token): Boolean = {
-      isSoftKw(token, name) && dialect.allowSignificantIndentation
-    }
-  }
-
-  @classifier
-  trait SkInfix {
-    val name = "infix"
-    def unapply(token: Token): Boolean = {
-      isSoftKw(token, name) && dialect.allowInfixMods
-    }
-  }
-
```

Comment:
why not extract all the other classifiers, too? lots of them are obviously useful to like whitespace, trivia etc.

Reply:
Most of it is internal to parser, so to be honest I am not sure what would make sense to extract. I will start with the soft keywords now, but let me know if you find anything that might be particularily useful in scalafmt.